support/proxy/vwf.example.com/mil-sym/unitSimData.vwf.yaml

```diff
@@ -0,0 +1,37 @@
+--- 
+extends: http://vwf.example.com/node.vwf
+properties:
+  posture: 
+    set: |
+      if ( this.posture !== value ) {
+        this.posture = value;
+      }
+    value: 'standing'
+
+  formation: 
+    set: |
+      if ( this.formation !== value ) {
+        this.formation = value;
+      }
+    value: 'wedge'
+
+  weaponStatus: 
+    set: |
+      if ( this.weaponStatus !== value ) {
+        this.weaponStatus = value;
+      }
+    value: 'weaponsTight'
+
+  pov: 
+    set: |
+      if ( this.pov !== value ) {
+        this.pov = value;
+      }
+    value: false
+
+events:
```

Comment:
No need to include `events`, `methods`, or `scripts` if they are not used.  :smile: